### ISSUE TITLE
Preview and Responsive CSS Grid Librarys

### ISSUE THREAD

**Issue by Triggsy22** [WordPress/gutenberg#24891]
Hello,

my Agency uses Gutenberg with only Custom Blocks. Currently we have over 80 of them.
All of them are based on BootstrapV4 and his Gridsystem

With Wordpress 5.5 the Preview is completely broken.
Here an Example:

Desktop Preview
![1](https://user-images.githubusercontent.com/33725296/91562217-1e5e6100-e93d-11ea-9550-4dbaa895005c.PNG)

Smartphone Preview
![2](https://user-images.githubusercontent.com/33725296/91562230-24544200-e93d-11ea-8b72-900638bc088a.PNG)

The Problem is, that the CSS media max-width, min-width is not working in the Preview. How could it? Its not a Page in itself, no iframe or something.

I read something about a possibility to fix this: https://make.wordpress.org/core/2020/08/04/new-editor-preview-options/
But in our Situation its nearly impossible to use. We cant rebuild the whole Bootstrap Grid System, to work in this way.

I think the only working Solution is the Preview in an Iframe, and we already discussed this: #20797

Best Regards.

**Comment by mtias** [comment]
Agreed on the difficulties in re-implementing certain responsive features. Progress here is blocked on making the iframe experiment work or switching preview to be non-editable.

**Comment by hawkmusician** [comment]
You can also implement conditional style tags that take care of the different preview breakpoints (previewDeviceType). That's what we're doing our side and it's working pretty well. To be honest, the non-editable preview seems a no-go for me in this day and age. 
The iframe would be a nice alternative, but the current option works perfectly, nice performance wise, and fun once you understand how it works (which is pretty simple).

**Comment by jordesign** [comment]
Hi @Triggsy22   - just checking through older issues and wanted to see if this is still relevant for your current use case?

**Comment by Triggsy22** [comment]
> Hi @Triggsy22 - just checking through older issues and wanted to see if this is still relevant for your current use case?

Its a long time ago. 
I dont know if this bug still exists. We are not using Gutenberg anymore.
So for us its not relevant anymore.

**Comment by jordesign** [comment]
Thanks @Triggsy22 As there hasn't been much further input in 3 years - I'll close it now. If it proves to be reportable problem in the future we can start with a new issue.
